src/pages/AboutYou.js

```diff
@@ -0,0 +1,36 @@
+const { dashboardStyles } = require('../styles.js')
+const { html } = require('../utils.js')
+const Layout = require('../components/Layout.js')
+const LogoutLink = require('../components/LogoutLink.js')
+const SummaryTable = require('../components/SummaryTable.js')
+const ButtonLink = require('../components/ButtonLink.js')
+
+const aboutYouRows = ({ name, address }) => {
+  return [{ key: 'Name', value: name }, { key: 'Mailing address', value: address }]
+}
+
+const AboutYou = ({ data = {} }) =>
+  html`
+    <${Layout}>
+      <div class=${dashboardStyles}>
+        <${LogoutLink} />
+        <h1>About you</h1>
+        <p>
+          This is the current name and address we have on file for you. Please update any
```

Comment:
Are we putting the marital and dependents information on a different screen?

Reply:
Nope, it's the same screen.

![image](https://user-images.githubusercontent.com/2454380/56605306-d5bcc800-65d1-11e9-9ab8-8a1b572a0654.png)